description/dvd.shacl.ttl

```diff
@@ -0,0 +1,146 @@
+    dct:description """Datamodel voor de gedetailleerde beschrijving van dvd-objecten en hun componenten."""@nl ;
+    pav:version "0.0.1" .
+
+haDvd:DVDNodeShape a sh:NodeShape ;
+    sh:targetClass haDVD:DVD ;
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path dct:format ;
+        sh:hasValue "video" ;
+
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+
+        sh:name "format"@en ;
+        sh:name "format"@fr ;
+        sh:name "formaat"@nl ;
+
+        sh:description "Extra indication of the type of Video."@en ;
+        sh:description "Indication supplémentaire du type de Video."@fr ;
+        sh:description "Indicatie van het type Video."@nl ;
+
+        sh:message "the object of dct:format is not 'video', is missing or occurs more than once"@en ;
+
+        sh:severity sh:Violation 
+    ] ,
+    [
+        a sh:PropertyShape ;
+        sh:path rel:hsp ;
+        sh:class haDVD:DVDChapter ;
+
+        sh:minCount 0 ;
+
+        sh:name "has part"@en ;
+        sh:name "a une partie"@fr ;
+        sh:name "heeft een onderdeel"@nl ;
+
+        sh:description "Indicates the DVD chapters that make up the DVD as a whole."@en ;
+        sh:description "Indique les chapitres du DVD qui composent le DVD dans son ensemble."@fr ;
+        sh:description "Aanduiding van de dvd-hoofdstukken die deel uitmaken van de dvd als geheel."@nl ;
+
+        sh:message "the object of schema:hasPart is no DVDChapter"@en ;
+
+        sh:severity sh:Violation
+    ] .
+
+haDvd:DVDChapterNodeShape a sh:NodeShape ;
+    sh:targetClass haDVD:DVDChapter ;
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path dct:format ;
+        sh:hasValue "video" ;
+
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+
+        sh:name "format"@en ;
+        sh:name "format"@fr ;
+        sh:name "formaat"@nl ;
+
+        sh:description "Extra indication of the type of Video."@en ;
+        sh:description "Indication supplémentaire du type de Video."@fr ;
+        sh:description "Indicatie van het type Video."@nl ;
+
+        sh:message "the object of dct:format is not 'video', is missing or occurs more than once"@en ;
+
+        sh:severity sh:Violation 
+    ] ,
+    [
+        a sh:PropertyShape ;
+        sh:path rel:isi ;
+        sh:class haDVD:DVD ;
+
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+
+        sh:name "is part of"@en ;
+        sh:name "fait partie de"@fr ;
+        sh:name "is onderdeel van"@nl ;
+
+        sh:description "Indicates the DVD that the chapter is part of."@en ;
+        sh:description "Indique le DVD dont le chapitre fait partie."@fr ;
+        sh:description "Aanduiding van de dvd waarvan het hoofdstuk deel uitmaakt."@nl ;
+
+        sh:message "the object of schema:isPartOf is no DVD, is absent or occurs more than once"@en ;
+    ] ,
+    [
+        a sh:PropertyShape ;
+        sh:path haDVD:chapterNumber ;
```

Comment:
I was already hesitant to add this specific property, since you could/will also have the numbering of chapters in e.g. a book, so `schema:position` works and we could use it in the future for similar cases.

Only question remaining then: do we also replace the pageNumber-property we introduced for newspaper pages by this property?

Reply:
good question. I think we discussed it in that context already and decided against it, although I can't remember why :) pageNumber can also be a subproperty of schema:position and so can chapterNumber

Reply:
Alright, I added chapterNumber as a subproperty of schema:position.